Address typos (#8521)
magento/devdocs: src/guides/v2.3/extension-dev-guide/cache/page-caching/private-content.md

The are some exception cases:

-  Store and website switching, after any of these action customer section `cart` will be updated.

> -  Customer cart lifetime option `section_data_lifetime` which is 60 minutes by default. After scheduled time passes section `cart` will be updated.

Reviewer: ```suggestion
-  Customer cart lifetime option `section_data_lifetime` which is 60 minutes by default. After scheduled time passes, section `cart` will be updated.
```

> -  Customer cart lifetime option `section_data_lifetime` which is 60 minutes by default. After scheduled time passes, section `cart` will be updated.

Reviewer 2: ```suggestion
-  Customer cart lifetime option `section_data_lifetime` which is 60 minutes by default. After the scheduled time passes, the `cart` section updates.
```
